src/web/modules/users/deleteUserFromService.njk

```diff
@@ -0,0 +1,29 @@
+{% from "govuk/components/input/macro.njk" import govukInput %}
+{% from "govuk/components/button/macro.njk" import govukButton %}
+{% extends "layout/layout.njk" %}
+
+{% block main %}
+  <h1 class="govuk-heading-m">Remove user</h1>
+  <div id="confirm-remove-user-dialog">
+    <div class="govuk-error-summary govuk-!-margin-top-6" role="group" aria-labelledby="error-summary" tabindex="-1">
+      <h2 class="govuk-error-summary__title" id="error-summary">
+        Are you sure you want to remove {{ payUser.email }} from {{ service.name }}?
+      </h2>
+      <form id="remove-team-member-form" method="post" action="/users/{{ userId }}/service/{{ serviceId }}/delete">
```

Comment:
You could leave off the `action` attribute if the form is going to POST to the same URL as the URL to GET the page.